### ISSUE TITLE
Is there a way to disable <OutputPath> replacement?

### ISSUE THREAD

**Issue by NickCraver** [dotnet/sdk#550]
I'm attempting to use the new project system on a MVC5 application:
- Repo here: [MiniPofiler v4 \ Samples.Mvc5.csproj](https://github.com/MiniProfiler/dotnet/blob/309ec37f971c69b41e6c89f9aaf72a5635e6b7b6/samples/Samples.Mvc5/Samples.Mvc5.csproj)
- Main issue here: https://github.com/dotnet/cli/issues/4952

The problem is that MVC5 applications expect for `\bin` to be `\bin`, not `\bin\$(TargetFramework)`. Unfortunately, `Microsoft.NET.TargetFrameworkInference.targets` unconditionally appends the target framework here: [Microsoft.NET.TargetFrameworkInference.targets#L95](https://github.com/dotnet/sdk/blob/master/src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.TargetFrameworkInference.targets#L95)

```xml
  <PropertyGroup>
    <IntermediateOutputPath>$(IntermediateOutputPath)$(TargetFramework)\</IntermediateOutputPath>
    <OutputPath>$(OutputPath)$(TargetFramework)\</OutputPath>
  </PropertyGroup>
```
Is there any way to opt out of this behavior? It's definitely not desirable in some situations and breaks an otherwise mostly-working project system for people migrating. There are many benefits of a simpler `.csproj` (fewer 3-way merges and reloads alone) even before people migrate to .NET Core, etc. So I'd love to see this work.

**Comment by NickCraver** [comment]
For anyone else hitting this, I'm temporarily using this workaround:

```xml
<PropertyGroup>
  <PostBuildEvent>
  xcopy /s /y "$(MSBuildProjectDirectory)\bin\net462\*.*" "$(MSBuildProjectDirectory)\bin"
  </PostBuildEvent>
</PropertyGroup>
```

**Comment by NickCraver** [comment]
The new workaround for this is a `Directory.build.targets` file instead of a copy:

```xml
<Project>
     <PropertyGroup>
         <OutputPath>bin/</OutputPath>
         <OutDir>bin/</OutDir>
     </PropertyGroup>
 </Project> 
```

While `OutDir` affects most things, `OutputPath` is needed for the main project's PDB, e.g. `Samples.Mvc5.pdb`.

**Comment by nguerrera** [comment]
This was fixed by #811, which now allows you to set `<AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>` in your project.

**Comment by NickCraver** [comment]
@nguerrera Thanks, the new property is working great!
